index.html

```diff
@@ -96,14 +133,21 @@ <h1>Multi-Post
                </ul>
                <div class="external_link">
                    <a href="#">See Project</a>
-               </div>
+               
+              </div>
               </footer>
   
             
           </section>
           <section id="p3">
             <div id="introd_3">
+              
+                <div class="row">
+                  <div class="column">
+      
                   <img  class="project_image" src="./images/Snapshoot Portfolio (2).svg" alt="snapshoot image project" border-raduis:8px>
+                  </div>
+                  <div class="column" >
                   <h1>Tonic</h1>
                   <div id="general_inf_2">
                       <p class="title">CANOPY</p>
```

Comment:
As requested by the previous reviewer, kindly make sure to make your project's images intersect between left and right, as displayed below 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/95243996/201197659-279838ca-55cb-4b8b-949f-423e2aecad5b.png"/>  | <img src="https://user-images.githubusercontent.com/95243996/201198844-9dfd5f22-7c3b-4fed-9c83-09500fdb1351.png" />  |